src/app/InteractionModelEngine.h

```diff
@@ -100,7 +100,7 @@ class InteractionModelEngine : public Messaging::ExchangeDelegate
 
     /**
      *  Retrieve a CommandSender that the SDK consumer can use to send a set of commands.  If the call succeeds,
-     *  the consumer is responsible for calling Shutdown() on the CommandSender once it's done using it.
+     *  see CommandSender documentation for lifetime handling.
```

Comment:
nit: redirecting from a doc to another doc is generally somewhat inconveniant. Don't have a better suggestion though beside some more invasive changes that would not require documentation to figure out lifetime (smart-pointer/handle usage).

Reply:
Yeah, I was not too happy about this either, not least because it's not pointing directly at the relevant docs over there, but if I did that I'd be pointing to a function name that can also change...  I couldn't find a nicer solution just with documentation, unfortunately.  :(